src/components/TimeseriesChart/TimeseriesChart.tsx

```diff
@@ -19,8 +19,8 @@ export interface TimeseriesChartProps {
   panelHeight: number;
   timeseriesIds: number[];
   pointsPerSeries: number;
-  start: number;
-  end: number;
+  start: number | Date;
```

Comment:
Will you also document the props to the user (can be a UNIX ts or a Date object)?

Reply:
Yes, will do that as part of documenting the component as a separate issue